staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -92,6 +94,11 @@ func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	if !handlingInfo.serviceAvailable {
+		http.Error(w, fmt.Sprintf("service: %s/%s is unavailable", handlingInfo.serviceNamespace, handlingInfo.serviceName), http.StatusServiceUnavailable)
```

Comment:
do we want to expose the name/namespace of the backing service here?

Reply:
Expose the name/namespace of the backing service here will let the client know clearly which service is wrong.

What do you think? @sttts @deads2k

Reply:
the client of the API group doesn't generally get that info. I'd limit the information returned to the API group and version